bgpd/bgp_fsm.c

```diff
@@ -603,7 +603,7 @@ int bgp_update_delay_active(struct bgp *bgp)
 
 int bgp_update_delay_configured(struct bgp *bgp)
 {
-	if (bgp->v_update_delay)
+	if (bgp && bgp->v_update_delay)
```

Comment:
How is this possible to call bgp_update_delay_configured with a null value.  If we have gotten here with a null bgp value we have made a mistake in our coding, I suspect you added this code as a check for the SA warning, but frankly the check for peer->bgp being null is wrong in bgp_fsm_change_status